projects/microphysics/scripts/piggy-back.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import numpy as np
+import sys
+import xarray as xr
+import vcm
+import vcm.catalog
+import vcm.fv3.metadata
+import wandb
+import fv3viz
+
+import matplotlib.pyplot as plt
+
+from fv3fit.tensorboard import plot_to_image
+
+
+run_artifact_path = sys.argv[1]
+
+job = wandb.init(
+    job_type="piggy-back", project="microphysics-emulation", entity="ai2cm"
+)
+
+
+def get_url_wandb(artifact: str):
+    art = job.use_artifact(artifact)
+    path = "fv3config.yml"
+    url = art.get_path(path).ref
+    return url[: -len("/" + path)]
+
+
+url = get_url_wandb(run_artifact_path)
+wandb.config["run"] = url
+grid = vcm.catalog.catalog["grid/c48"].to_dask()
+piggy = xr.open_zarr(url + "/piggy.zarr")
+
+ds = vcm.fv3.metadata.gfdl_to_standard(piggy).merge(grid)
+ds["time"] = np.vectorize(vcm.cast_to_datetime)(ds.time)
+
+metrics = {}
+
+for field in ["cloud_water", "specific_humidity", "air_temperature"]:
+    emulator_var = f"tendency_of_{field}_due_to_zhao_carr_emulator"
+    physics_var = f"tendency_of_{field}_due_to_zhao_carr_physics"
+
+    fv3viz.plot_cube(ds.isel(time=0, z=50), emulator_var)
```

Comment:
Might be useful in the future to be able to run this for more than one level or time.  Not necessary for current PR though.